File: src/main/java/pt/joaomneto/titancompanion/adventurecreation/impl/COTSAdventureCreation.java
package pt.joaomneto.titancompanion.adventurecreation.impl;

		import java.io.BufferedWriter;
		import java.io.IOException;

public class COTSAdventureCreation extends BaseAdventureCreation {


	@Override
	protected void storeAdventureSpecificValuesInFile(BufferedWriter bw) throws IOException {
		bw.write("gold=0\n");
	}

}

File: src/main/java/pt/joaomneto/titancompanion/adventurecreation/impl/COTSAdventureCreation.kt
package pt.joaomneto.titancompanion.adventurecreation.impl

import java.io.BufferedWriter
import pt.joaomneto.titancompanion.adventurecreation.AdventureCreation

class COTSAdventureCreation : AdventureCreation() {
    override fun storeAdventureSpecificValuesInFile(bw: BufferedWriter) {
        super.storeAdventureSpecificValuesInFile(bw)
        bw.write("provisions=0\n")
        bw.write("provisionsValue=4\n")
    }
}
